src/charm.py

```diff
@@ -179,6 +180,7 @@ def _on_update_status(self, event):
         if not all_valid_plans:
             return
 
+        self.unit.set_workload_version(f"v{AIRBYTE_VERSION}")
```

Comment:
Didn't know about this, that's cool

Reply:
Yup, useful for informing users of the version being deployed when running `juju status`.